test/e2e/snaps/ppom-toggle-settings.spec.js

```diff
@@ -0,0 +1,75 @@
+const { strict: assert } = require('assert');
+const {
+  withFixtures,
+  unlockWallet,
+  openDapp,
+  defaultGanacheOptions,
+  getWindowHandles,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+describe('PPOM Settings', function () {
+  it('should not show the PPOM warning when toggle is off', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToTestDapp()
+          .build(),
+        ganacheOptions: defaultGanacheOptions,
```

Comment:
ganache default options use chainId 1337. These tests should be conducted using chainId 1, since it's where the PPOM functionality is active